framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -215,5 +218,8 @@ SideSetsAroundSubdomain::modify()
 
   // Assign the supplied names to the newly created side sets
   for (unsigned int i = 0; i < boundary_ids.size(); ++i)
+  {
     boundary_info.sideset_name(boundary_ids[i]) = _boundary_names[i];
+    boundary_info.nodeset_name(boundary_ids[i]) = _boundary_names[i];
```

Comment:
This is a fairly substantial change. Do we really want to stick these names into node sets? The documentation string here isn't accurate with this change.

Reply:
Several other side set modifiers already did this. The rationale is that since libMesh will add the new node set anyway along with the side set, it is better to give it a name. Otherwise in ParaView, you will see unnamed new node set while the new side set is named. Are you talking about the param doc string?

Reply:
I still don't think this is desired. If you turn on adaptivity you certainly don't want to use the nodeset to apply boundary conditions. If other modifiers are doing this, they should probably be cleaned up. I'd be OK naming the nodeset but I believe that the names should be different. I believe MOOSE prefers sidesets over nodesets in code but we should be careful about using them interchangeably.

Reply:
Still don't like this. Maybe @aeslaughter can chime in?

Reply:
I actually agree if we can let libMesh only generate side sets as the name of the modifier suggests. If user wants to add node sets, they can do that from existing side sets.

Reply:
@permcody I am not sure why this change is being made, I probably need more clarification to why this is needed.

The issue this closes states:
> It will be nice this modifier to support a union of subdomains. Plus the new behavior cannot be produced with the combination of current modifiers.

I am not sure how changing adding nodeset relates.

Reply:
Right, I think we've gotten sloppy with sidesets and nodesets in MOOSE and I'd like to stop the propagation of the mess. I know Yaqi is correct in that some of the other modifiers haphazardly assign nodeset and sideset names to boundaries that they create. MOOSE even handles this OK since we prefer sidesets over nodesets when they exist. I just think that it's going to get us in trouble.

Reply:
Should we create a separate issue like 'Do not add node set when adding side set'? I can trim this change off. I just feel we need to be consistent and paraview showing an unnamed node set while all others are paired up nicely.

Reply:
Yes, let's add a separate issue. I'd like to see this trimmed off. We can go back and start removing these in the other modifiers.